com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/HDBakedReflectionSystem.cs

```diff
@@ -635,7 +635,7 @@ internal static void ImportAssetAt(HDProbe probe, string file)
                         importer.sRGBTexture = false;
                         importer.filterMode = FilterMode.Bilinear;
                         importer.mipmapEnabled = false;
-                        importer.textureCompression = hd.currentPlatformRenderPipelineSettings.lightLoopSettings.planarReflectionCacheCompressed
+                        importer.textureCompression = false
```

Comment:
Could just set to uncompressed right?

Reply:
woops didn't see the part of the lines after, my bad